{
  "uid" : "42985c18ac6d8df8",
  "name" : "[US-111]/[3] I can't log into automationpractice.com using email \"thor.odinson@example.com\" & password \"null\"",
  "fullName" : "tests.WebAPP.Authentication_Tests.test_3",
  "historyId" : "b99540ace0d77e3fb356d761a59ccb9f",
  "time" : {
    "start" : 1591459416144,
    "stop" : 1591459421240,
    "duration" : 5096
  },
  "description" : "[US-111]/[2] As a user I can't log into automationpractice.com using email \"thor.odinson@example.com\" & password \"null\"",
  "descriptionHtml" : "<p>[US-111]/[2] As a user I can't log into automationpractice.com using email &quot;thor.odinson@example.com&quot; &amp; password &quot;null&quot;</p>\n",
  "status" : "failed",
  "statusMessage" : "Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
  "statusTrace" : "java.lang.AssertionError: Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_3(Authentication_Tests.java:100)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
  "flaky" : false,
  "newFailed" : true,
  "beforeStages" : [ {
    "name" : "Setting up Test Class",
    "time" : {
      "start" : 1591459415365,
      "stop" : 1591459416138,
      "duration" : 773
    },
    "description" : "Setting up Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Setting up Excel File",
    "time" : {
      "start" : 1591459258563,
      "stop" : 1591459261011,
      "duration" : 2448
    },
    "description" : "Setting up Excel File",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "[US-111]/[2] As a user I can't log into automationpractice.com using email \"thor.odinson@example.com\" & password \"null\"",
    "status" : "failed",
    "statusMessage" : "Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
    "statusTrace" : "java.lang.AssertionError: Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_3(Authentication_Tests.java:100)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
    "steps" : [ {
      "name" : "I open home page",
      "time" : {
        "start" : 1591459416152,
        "stop" : 1591459419305,
        "duration" : 3153
      },
      "status" : "failed",
      "statusMessage" : "Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "statusTrace" : "java.lang.AssertionError: Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_3(Authentication_Tests.java:100)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "e4978bde3f6c0eb4",
      "name" : "Scenario FAIL screenshot",
      "source" : "e4978bde3f6c0eb4.png",
      "type" : "image/png",
      "size" : 20127
    }, {
      "uid" : "98c935b616e0f0c2",
      "name" : "TestNG test FAIL logs",
      "source" : "98c935b616e0f0c2.txt",
      "type" : "text/plain",
      "size" : 2919
    } ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Teardown Test Class",
    "time" : {
      "start" : 1591459421241,
      "stop" : 1591459421339,
      "duration" : 98
    },
    "description" : "Teardown Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Teardown Test Suite",
    "time" : {
      "start" : 1591459632288,
      "stop" : 1591459633275,
      "duration" : 987
    },
    "description" : "Teardown Test Suite",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testClass",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testMethod",
    "value" : "test_3"
  }, {
    "name" : "parentSuite",
    "value" : "Automationpractice.com WEB - Kamil Nowocin"
  }, {
    "name" : "suite",
    "value" : "Automated tests for WEB client"
  }, {
    "name" : "subSuite",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "host",
    "value" : "travis-job-b8e8384f-8b1f-4752-b45a-8c76077aa547"
  }, {
    "name" : "thread",
    "value" : "6311@travis-job-b8e8384f-8b1f-4752-b45a-8c76077aa547.TestNG-test=Automated tests for WEB client-4(18)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "owner",
    "value" : "Kamil Nowocin"
  }, {
    "name" : "story",
    "value" : "NEGATIVE FLOW"
  }, {
    "name" : "feature",
    "value" : "LOGIN TESTS"
  }, {
    "name" : "epic",
    "value" : "Web App Tests"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "TAP-0003",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/issue/TAP-0003",
    "type" : "issue"
  }, {
    "name" : "STORY-111",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/STORY-111",
    "type" : "tms"
  } ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "owner" : "Kamil Nowocin",
    "retries" : [ {
      "uid" : "6b87118179d26534",
      "status" : "skipped",
      "statusDetails" : "Upssss, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "time" : {
        "start" : 1591459414925,
        "stop" : 1591459415280,
        "duration" : 355
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 1,
        "broken" : 1,
        "skipped" : 14,
        "passed" : 105,
        "unknown" : 0,
        "total" : 121
      },
      "items" : [ {
        "uid" : "2ea333607c24804c",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/170093163/#testresult/2ea333607c24804c",
        "status" : "passed",
        "time" : {
          "start" : 1591457828522,
          "stop" : 1591457840834,
          "duration" : 12312
        }
      }, {
        "uid" : "227cf84963e19087",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/170092304/#testresult/227cf84963e19087",
        "status" : "passed",
        "time" : {
          "start" : 1591456997132,
          "stop" : 1591457011152,
          "duration" : 14020
        }
      }, {
        "uid" : "e9181d6e0307c109",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/170090935/#testresult/e9181d6e0307c109",
        "status" : "passed",
        "time" : {
          "start" : 1591455486434,
          "stop" : 1591455501411,
          "duration" : 14977
        }
      }, {
        "uid" : "46c55c06906ac605",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/170085385/#testresult/46c55c06906ac605",
        "status" : "passed",
        "time" : {
          "start" : 1591449679885,
          "stop" : 1591449692702,
          "duration" : 12817
        }
      }, {
        "uid" : "fc02f7082969d39b",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/170083871/#testresult/fc02f7082969d39b",
        "status" : "passed",
        "time" : {
          "start" : 1591447553100,
          "stop" : 1591447563545,
          "duration" : 10445
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "42985c18ac6d8df8.json",
  "parameterValues" : [ ]
}